ember-cli-build.js

```diff
@@ -20,78 +20,100 @@ module.exports = function (app) {
     })
   ]);
 
+  const qunit = new Funnel(path.dirname(require.resolve('qunit')), {
+    annotation: 'tests/qunit.{js,css}',
+    destDir: 'tests',
+    files: ['qunit.css', 'qunit.js']
+  });
+
+  const loader = new Funnel(path.dirname(require.resolve('loader.js')), {
+    annotation: 'tests/loader.js',
+    destDir: 'tests',
+    files: ['loader.js']
+  });
+
+  const tests = new Funnel(__dirname + '/tests', {
+    destDir: 'tests',
+    files: ['index.html']
+  });
+
   const compiled = typescript(src, {
     throwOnError: process.env.EMBER_ENV === 'production',
   });
 
+  const compiledDeclarations = typescript('lib', {
+    tsconfig: {
```

Comment:
In general we shouldn't need to pass `tsconfig` here, it should just be able to use our top level tsconfig.json.

Reply:
If we don't explicitly flag for `declaration` here and default to `tsconfig` with `declaration: true` we will have to cherry-pick out the `.d.ts` files for tests.

Reply:
Actually this brings up a good point as to; do we actually need to output tests within `dist`?

Reply:
^^^^ @krisselden